index.js

```diff
@@ -449,8 +532,8 @@ const dialogFactory = (core, proc, win) => {
     action(() => vfs.unlink(file, {pid: proc.pid}), true, __('MSG_DELETE_ERROR'));
   }));
 
-  const progressDialog = (file) => dialog('progress', {
-    message: __('DIALOG_PROGRESS_MESSAGE', file.name),
+  const progressDialog = name => dialog('progress', {
```

Comment:
@andersevenrud there's mixing and matching of whether single parameters should have parenthesis around them in this file, is there one you would prefer to standardize to?

Reply:
@ajmeese7 I don't *really* have a preference, but kinda pivoting for always having parentheses for function expressions 🤷‍♂️

Reply:
Agreed. I changed the code style